sdk/anomalydetector/Azure.AI.AnomalyDetector/TempCadlFiles/cadl/cadl-project.yaml

```diff
@@ -0,0 +1,17 @@
+emitters:
```

Comment:
These files should not be here.  I wonder if there is a path mismatch when using service.proj vs pointing at a project directory?  We will need to investigate this.